.rubocop.yml

```diff
@@ -4,69 +4,55 @@ AllCops:
   Exclude:
     - 'db/schema.rb'
     - 'vendor/**/*'
-  TargetRubyVersion: 2.1
+    - 'Gemfile'
+  TargetRubyVersion: 2.2
```

Comment:
2.2? Any reason not to target 2.4?

Reply:
Other than 2.4 increasing the rubocop errors 2 fold...
The Travis builds and tests 2.2, 2.3, and 2.4 so technically that seems to be our oldest supported version